### DISCUSSION TITLE
Bug Undo/Redo

### DISCUSSION THREAD

**Discussion by khanhleemtp** [GrapesJS/grapesjs#4319] [Q&A] [answered]
When i using events of editor,
```javascript
editor.on('component:selected', comp => {
// todo st
})
```

- I can't undo exactly
- Help me, please....

**Comment by ronaldohoch** [comment, marked as answer]
We set `undoManager: { trackSelection: false },` on `editor.init` to stop tracking the selection.
My question: https://github.com/artf/grapesjs/discussions/4226
